### PR TITLE
Test cmds repair

### PR DESCRIPTION
**Pull request by Ernold11**

### CODE REVIEW

**Comment by emcek** [comment]
Question, what was wrong, why we need those changes?

File: moler/util/cmds_doc.py [hunk 1]
```diff
@@ -49,8 +49,9 @@ def _walk_moler_python_files(path):
     :type path:
     :rtype: str
     """
-    repo_path = abspath(join(dirname(__file__), '../..'))
-    path = join(__file__.partition('moler/util/cmds_doc.py')[0], path)
+    repo_path = abspath(join(path, '../..'))
```
Comment (Ernold11) [inline moler/util/cmds_doc.py]: Please use os.path.join, remove print

File: moler/util/cmds_doc.py [hunk 2]
```diff
@@ -187,7 +188,9 @@ def check_if_documentation_exists(path2cmds):
     wrong_commands = {}
     errors_found = []
     print()
+    found_any_command = False
```
Comment (Ernold11) [inline moler/util/cmds_doc.py]: Change from flag into int and on the end print number of tested commands

File: moler/util/cmds_doc.py (hunk 1 above)
Comment (Ernold11) [reply moler/util/cmds_doc.py]: I leave print for debug reason. If path will be correct print can be remove.

**Pull request merged**